tutorials/step_by_step/simple_2d_game.rst

```diff
@@ -107,6 +107,7 @@ and the pad.
     # Member variables
     var screen_size
     var pad_size
+    var direction = Vector2(1.0, 0.0)
```

Comment:
It should then be added to all other code snippets which show member variables below.